echopype/echodata/convention/utils.py

```diff
@@ -0,0 +1,16 @@
+from echopype.echodata.convention import sonarnetcdf_1
```

Comment:
The above method that I'm using is more explicit, but I can go with your suggestion.

Reply:
If you use this more explicit solution, then it will not work if you are trying to run straight from the codebase.

Reply:
I actually like the more explicit. So I'm going to resolve this conversation.

Reply:
It is actually recommended to be more explicit. https://peps.python.org/pep-0008/#imports

Things should work. You have to do `pip install -e .`

Reply:
I think what you're discussing here is absolute vs relative imports, not "explicit" vs less explicit. @lsetiawan thanks for that helpful pep8 link. What I'm reading there is that explicit is always preferred (no surprise), absolute imports are recommended, but relative imports are an acceptable alternative. Also, I think relative imports is the most common practice in echopype, so for code style consistency I would suggest relative imports (@b-reyes 's suggestion)

Reply:
Alright. 😛 Though I think it would be good for echopype to update this in the near future to be a more absolute imports for better error messaging, as PEP8 nicely put it:
> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages)

Reply:
I agree, we should clean this up in the future.